conf/messages.en

```diff
@@ -470,6 +470,7 @@ reportingPeriod.first.rpo.p1 = This is the first year the platform operator beca
 reportingPeriod.first.epo.p1 = This is the first year the platform operator became an excluded platform operator. Each reportable period is a calendar year that runs from 1 January to 31 December.
 reportingPeriod.first.hint = This is a year, for example 2024.
 reportingPeriod.error.required = Enter the reportable period
+reportingPeriod.error.nonNumeric = Year must include 4 numbers
```

Comment:
Shiould this say "Year must be 4 digits"

Reply:
This message is from Kerry so I would leave it.